src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+import './styles.css';
+import React from 'react';
+
+const Calculator = () => (
+  <div className="calculator-grid">
+    <div className="output">0</div>
+    <button type="button">AC</button>
+    <button type="button">+/-</button>
+    <button type="button">%</button>
+    <button className="btn-orange" type="button">/</button>
+    <button type="button">7</button>
+    <button type="button">8</button>
+    <button type="button">9</button>
+    <button className="btn-orange" type="button">*</button>
+    <button type="button">4</button>
+    <button type="button">5</button>
+    <button type="button">6</button>
+    <button className="btn-orange" type="button">-</button>
+    <button type="button">1</button>
+    <button type="button">2</button>
+    <button type="button">3</button>
+    <button className="btn-orange" type="button">+</button>
+    <button type="button" className="span-two">0</button>
+    <button type="button">.</button>
+    <button className="btn-orange" type="button">=</button>
+
+  </div>
+);
```

Comment:
Nice work on the calculator component :+1: However, as requested in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component), you should create another component that will be used as a child component in `Calculator`.

Reply:
@laiifuu Done, Thanks for the review.